cardano-db-sync/src/Cardano/DbSync/Era/Shelley/Genesis.hs

```diff
@@ -193,10 +196,10 @@ insertTxOuts blkId (address, value) = do
 -- -----------------------------------------------------------------------------
 
 configGenesisHash :: ShelleyGenesis TPraosStandardCrypto -> ByteString
-configGenesisHash = const "Hash of genesis configuration"
+configGenesisHash _ = Shelley.fakeGenesisHash
 
 genesisHashSlotLeader :: ShelleyGenesis TPraosStandardCrypto -> ByteString
-genesisHashSlotLeader = BS.take 28 . configGenesisHash
+genesisHashSlotLeader = BS.take 32 . configGenesisHash
```

Comment:
Eh. Maybe we have some existing functions for that in other repos? I hope.

Reply:
That is a placeholder for when real genesis hashing is implemented.